app/models/anime_search.rb

```diff
@@ -0,0 +1,40 @@
+class AnimeSearch
+  include ActiveModel::Model
+
+  SHOW_TYPES = %w(OVA ONA Movie TV Special Music)
+
+  SEASONS = %w(summer fall winter spring)
+
+  attr_accessor(
```

Comment:
Big lists of symbols like this should be aligned, and since `attr_accessible` is a DSL method, it shouldn't have parentheses.  Like so:

```ruby
attr_accessor :name,
              :slug,
              :show_type
```